SampleConsoleClient/Program.cs

```diff
@@ -128,13 +130,16 @@ private static void TestRESTInterface(string[] args)
             string[] keywords = { "*" }; // return everything
             string address = webClient.BaseAddress + "infomodel/find";
             webClient.Headers.Add("Content-Type", "application/json");
-            string response = webClient.UploadString(address, "PUT", JsonSerializer.Serialize(keywords));
-            Console.WriteLine(response);
+            var response = webClient.UploadString(address, "PUT", System.Text.Json.JsonSerializer.Serialize(keywords));
+            UANodesetResult[] identifiers = JsonConvert.DeserializeObject<UANodesetResult[]>(response);
+            for (var i = 0; i < identifiers.Length; i++)
+            {
+                Console.WriteLine(JsonConvert.SerializeObject(identifiers[i], Formatting.Indented));

```

Comment:
Why not simply use .ToString() here instead of converting back to JSON?

Reply:
Because in the next block of code we need the Id member. We have to parse the object to find that value.